packages/components/src/components/select/select.stories.ts

```diff
@@ -0,0 +1,427 @@
+const multipleConstant: ConstantDefinition = { type: 'attribute', name: 'multiple', value: true };
+const helpTextConstant: ConstantDefinition = { type: 'attribute', name: 'help-text', value: 'help-text' };
+const labelConstant: ConstantDefinition = { type: 'attribute', name: 'label', value: 'Label' };
+
+// Stories
+export default {
+  title: 'Components/sd-select',
+  component: 'sd-select',
+  args: overrideArgs([
+    threeOptionsConstant,
+    labelConstant,
+    { type: 'attribute', name: 'placeholder', value: 'Please Select' },
+    { type: 'attribute', name: 'max-options-visible', value: 3 }
+  ]),
+  argTypes: {
+    ...argTypes,
+    'value-attr': {
+      name: 'value',
+      table: {
+        category: 'attributes',
+        defaultValue: ''
+      },
+      description:
+        'The current value of the select, submitted as a name/value pair with form data. When `multiple` is enabled, the value attribute will be a space-delimited list of values based on the options selected, and the value property will be an array. **For this reason, values must not contain spaces.**',
+      control: 'text'
+    }
+  },
+  parameters: {
+    ...parameters,
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/ZphyVFsUHL72voMrJagMeo/Select?node-id=1002%3A2284&mode=dev'
+    }
+  }
+};
+
+/**
+ * This shows sd-select in its default state.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return html`<div class="h-[260px] w-[420px]">${generateTemplate({ args })}</div>`;
+  }
+};
+
+/**
+ * Use the `size` attribute to change a select’s size. It will cascade to slotted `sd-option` elements.
+ */
+
+export const SizeMultiple = {
+  name: 'Size x Multiple',
+  parameters: {
+    controls: {
+      include: []
+    }
+  },
+  render: () => {
+    return html`<div class="h-[340px]">
+      ${generateTemplate({
+        options: {
+          classes: 'w-full'
+        },
+        axis: {
+          x: {
+            type: 'attribute',
+            name: 'size'
+          },
+          y: {
+            type: 'attribute',
+            name: 'useTags',
+            values: [false, true]
+          }
+        },
+        constants: [
+          fiveOptionsConstant,
+          multipleConstant,
+          { type: 'attribute', name: 'value', value: 'option-1 option-2 option-3 option-4' }
+        ],
+        args: null
+      })}
+    </div>`;
+  }
+};
+
+/**
+ * To allow multiple options to be selected, use the `multiple` attribute. It’s a good practice to use `clearable` when this option is enabled. To use the checkbox with tags variant, set the `useTags` variant to `true`.  To set multiple values at once, set value to a space-delimited list of values.  The preferred placement of the select’s listbox can be set with the `placement` attribute. Note that the actual position may vary to ensure the panel remains in the viewport. Valid placements are `top` and `bottom`.
+ */
+
+export const DisabledMultiple = {
```

Comment:
I am not sure if this is a bug in the `sd-tag` of here but if the `sd-select` is disabled, we still have a hover effect on the selected option. This seems wrong. Or is this the expected design?

Reply:
this is indeed wrong